--- a/yabause/src/retro_arena/CMakeLists.txt
+++ b/yabause/src/retro_arena/CMakeLists.txt
@@ -1,4 +1,5 @@
 project(yabause-retro-arena)
+cmake_minimum_required(VERSION 3.5)

 add_definitions( -D__RETORO_ARENA__ )

@@ -134,7 +134,7 @@
 target_link_libraries(yabause-retro-arena yabause  ${Boost_LIBRARIES} pugixml ${YABAUSE_LIBRARIES} ${PORT_LIBRARIES} nanogui ${LIBCHDR_LIBRARIES} )
 target_link_libraries( yabause-retro-arena ${LIBCHDR_LIBRARIES} )
 target_link_libraries( yabause-retro-arena ${zlib_STATIC_LIBRARIES} )
 target_link_libraries( yabause-retro-arena ${CMAKE_CURRENT_BINARY_DIR}/lib/sqlite3.lib )
-add_dependencies(yabause-retro-arena Json png zlib libchd nanogui external_sqlite3)
+add_dependencies(yabause-retro-arena Json zlib nanogui external_sqlite3)
 yab_port_success(yabause-retro-arena)
 set_target_properties(yabause-retro-arena PROPERTIES OUTPUT_NAME yabasanshiro)

--- a/yabause/CMake/Packages/external_zlib.cmake
+++ b/yabause/CMake/Packages/external_zlib.cmake
@@ -69,6 +69,7 @@
 		-DCMAKE_POSITION_INDEPENDENT_CODE:BOOL=ON
         -DCMAKE_BUILD_TYPE:STRING=Release
         -DCMAKE_INSTALL_PREFIX:STRING=${ZLIB_INSTALL}
+        -DCMAKE_POLICY_VERSION_MINIMUM=3.5
         ${ADDITIONAL_CMAKE_ARGS}        
 )
 
--- a/yabause/CMake/Packages/external_libchdr.cmake
+++ b/yabause/CMake/Packages/external_libchdr.cmake
@@ -36,6 +36,7 @@
               -DCMAKE_BUILD_TYPE:STRING=Release 
               -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
               -DCMAKE_C_FLAGS=${CMAKE_C_FLAGS}
+              -DCMAKE_POLICY_VERSION_MINIMUM=3.5
               ${ADDITIONAL_CMAKE_ARGS}
 )
 
--- a/yabause/src/retro_arena/CMakeLists.txt
+++ b/yabause/src/retro_arena/CMakeLists.txt
@@ -59,7 +59,7 @@
      # For shallow git clone (without downloading whole history)
      #GIT_SHALLOW 1
      # For point at certain tag
-     GIT_TAG v3.2.0
+     GIT_TAG "359f98d14065bf4e53eeb274f5987fd08f16e5bf"
      #disables auto update on every build
      UPDATE_DISCONNECTED 1
      #disable following
